src/index.js

```diff
@@ -150,7 +202,7 @@ function clearSelection() {
 class App extends React.Component {
   componentDidMount() {
     this.onKeyDown = event => {
-      if (event.key === "Backspace") {
+      if (event.key === "Backspace" && event.target.nodeName !== "INPUT") {
```

Comment:
Do you know what’s the best practice to deal with key bindings like this? This works fine as a hack but I wonder if there’s a better solution (not for this PR)

Reply:
Not sure. In my projects I use a full-blown, global hotkey manager, which would be unsuitable here. We'll figure this out as we go along.

Reply:
Sounds good :)